### ISSUE TITLE
Function app failing when cloudflare logs blob are large

### ISSUE THREAD

**Issue by beachfires** [Azure/Azure-Sentinel#7808]
https://github.com/Azure/Azure-Sentinel/blob/e076a957cdda7af8365d7b8d22f8b0b7c0b2255d/Solutions/Cloudflare/Data%20Connectors/AzureFunctionCloudflare/main.py#L31

During an event where we were experiencing an influx of events on Cloudflare (DDoS) the function app responsible for processing these logs from the Storage account started failing. This resulted in days without logs as it kept attempting to process the same logs and failing repeatedly, effectively halting Cloudflare log ingestion.

The blob sizes in our storage container were reaching up to 83M in size (during the DDoS) vs 52K during normal circumstances.

I've tried to tune the parameters available to me, like `MAX_BUCKET_SIZE` and `MAX_CONCURRENT_PROCESSING_FILES` to no avail. This is the error we would receive when looking through application insights for the function app:

```Exception while executing function: Functions.AzureFunctionCloudflare Result: Failure
Exception: HttpResponseError: Download stream interrupted.
Stack:   File "/azure-functions-host/workers/python/3.8/LINUX/X64/azure_functions_worker/dispatcher.py", line 448, in _handle__invocation_request
    call_result = await self._run_async_func(
  File "/azure-functions-host/workers/python/3.8/LINUX/X64/azure_functions_worker/dispatcher.py", line 724, in _run_async_func
    return await ExtensionManager.get_async_invocation_wrapper(
  File "/azure-functions-host/workers/python/3.8/LINUX/X64/azure_functions_worker/extension.py", line 147, in get_async_invocation_wrapper
    result = await function(**args)
  File "/home/site/wwwroot/AzureFunctionCloudflare/main.py", line 59, in main
    await asyncio.gather(*cors)
  File "/home/site/wwwroot/AzureFunctionCloudflare/main.py", line 107, in process_blob
    async for chunk in blob_cor.chunks():
  File "/home/site/wwwroot/.python_packages/lib/site-packages/azure/storage/blob/aio/_download_async.py", line 151, in __anext__
    self._current_content = await self._iter_downloader.yield_chunk(chunk)
  File "/home/site/wwwroot/.python_packages/lib/site-packages/azure/storage/blob/aio/_download_async.py", line 62, in yield_chunk
    return await self._download_chunk(chunk_start, chunk_end - 1)
  File "/home/site/wwwroot/.python_packages/lib/site-packages/azure/storage/blob/aio/_download_async.py", line 105, in _download_chunk
    chunk_data = await process_content(response, offset[0], offset[1], self.encryption_options)
  File "/home/site/wwwroot/.python_packages/lib/site-packages/azure/storage/blob/aio/_download_async.py", line 27, in process_content
    raise HttpResponseError(message="Download stream interrupted.", response=data.response, error=error)
 ```

This is a critical issue as it affects us when we need it most, during a security incident. Could you please have a look at what improvements could be made to stop this from happening again.

**Comment by beachfires** [comment]
I just want to add a few more details about the errors we were having. Here is a screenshot from app insights:
<img width="527" alt="image" src="https://user-images.githubusercontent.com/72743072/231501477-6669ef9c-d5ea-4c30-a0cf-e10420eda2fe.png">

**Comment by sawp-d** [comment]
@beachfires Did you ever end up fixing this? I'm running into the same issue as well. 

@v-amolpatil Has this been addressed yet?

**Comment by v-vdixit** [comment]
Hi @beachfires, we already have a PR #7983 with the resolution of this issue, once the PR is merged the issue will be resolved, thanks!

**Comment by v-vdixit** [comment]
Hi @beachfires we are discussing with the solution owners the code has been updated the latest version of package will be published shortly, thanks!

**Comment by v-vdixit** [comment]
Hi @beachfires the issue has been resolved in the latest package of Cloudflare which was merged in this PR - #8217, we are closing this issue, if you need assistance with something else please feel free to reach out to us, thanks!

**Comment by Goutham051101** [comment]
Hello @v-amolpatil we have files more than 20 MB is it possible to process.
